contrib/completion/bash/docker-compose

```diff
@@ -158,6 +163,18 @@ _docker-compose_migrate-to-labels() {
 }
 
 
+_docker-compose_pause() {
```

Comment:
You have to take care of the non-boolean options here, otherwise `docker-compose pause --timeout` will complete service names. Add a `case "$prev" in` block here, see `_docker-compose_restart()` for an example.